src/services/exposed/scope-service.js

```diff
@@ -0,0 +1,18 @@
+const context = require('../../context');
+
+class ScopeService {
+  static async getScopes(renderingId) {
+    const { configStore, forestServerRequester } = context.inject();
+    const { envSecret } = configStore.lianaOptions;
```

Comment:
checkDependency?

Reply:
FES and FE have no `checkDependencies` implemented for now. It could be great to add it but a bit irrelevant through that PR